src/drivers/net/dwc_gmac/Mybuild

```diff
@@ -0,0 +1,19 @@
+package embox.driver.net
+
+module dwc_gmac {
+	option number log_level = 1
```

Comment:
I think it should be zero by default

Reply:
If set log_level=0 error message don't show by default, It's required for small platform.